src/core_atmosphere/physics/physics_wrf/module_ra_cam_support.F

```diff
@@ -8,7 +8,7 @@ MODULE module_ra_cam_support
   implicit none
       integer, parameter :: r8 = 8
       real(r8), parameter:: inf = 1.e20 ! CAM sets this differently in infnan.F90
-      integer, parameter:: bigint = O'17777777777'           ! largest possible 32-bit integer 
+      integer, parameter:: bigint = int(O'17777777777')      ! largest possible 32-bit integer 
```

Comment:
Would using the `huge` intrinsic be better here? GNU, Intel, PGI and Nag are cool with the following:
```
integer, parameter :: bigint = huge(bigint)
```

Reply:
That is a good suggestion and something that I had considered. However, I don't know that `huge(bigint)` will be equivalent to `int(O'17777777777')` on every machine (it depends on the machine's representation of integers, and I would agree that what the code is doing now is completely not portable), and I was concerned about inadvertently changing the value of `bigint`. In the end, I considered it better to modify the code as minimally as possible while getting it to comply with the Fortran 2003 standard.

Reply:
That's what I figured your rational was, sounds good to me!